plugin-server/src/worker/plugins/loadPlugin.ts

```diff
@@ -22,6 +22,11 @@ export async function loadPlugin(hub: Hub, pluginConfig: PluginConfig): Promise<
         return false
     }
 
+    if (plugin.plugin_type === 'inline') {
+        await pluginConfig.instance?.initialize!('', pluginDigest(plugin))
+        return true
+    }
```

Comment:
without this when trying to use the plugin I got
<img width="955" alt="Screenshot 2024-07-17 at 19 24 33" src="https://github.com/user-attachments/assets/767dbb76-6e9f-4a7d-a252-81a5219a4616">

this shouldn't be part of this PR, @oliverb123 feel free to incorporate the code directly or some other way.

Reply:
Sure, I'll pull it into the one this one is stack one, thanks for the heads up

Reply:
Ah, hm, maybe this isn't necessary at all - the word "config" is pretty overloaded in the code, but I guess this is referring to the "plugin.json", which inline plugins won't need - just early exiting is fine. I'll pull it into the PR below.

Reply:
yup this is just plugin related stuff (loading the necessary source files). Vs attatchments are pluginConfig related and done https://github.com/PostHog/posthog/blob/1d81f1af727842b76d8982b477acefa1c05a502d/plugin-server/src/worker/plugins/loadPluginsFromDB.ts#L43